models/base/pageviews_proc.sql

```diff
@@ -31,22 +28,13 @@ max(if(params.key = 'campaign', params.value.string_value, null)) utm_campaign,
 max(if(params.key = 'page_referrer', params.value.string_value, null)) utm_referrer,
 max(ecommerce.transaction_id) ecommerce_transaction_id,
 max(ecommerce.purchase_revenue) ecommerce_purchase_revenue
-FROM 
-
--- Check if _intraday or daily events tables should be used
-{% if intraday_tables|length > 0 %}
-	`{{ target.project }}.{{ target.schema }}.events_intraday_*`,
-{% else %}
-	`{{ target.project }}.{{ target.schema }}.events_*`,
-{% endif %}
-
+FROM
```

Comment:
Would we be double-counting sessions if we aggregate both intraday + daily?

My understanding is that they'd converge towards being identical datasets per day, lmk if that's incorrect.

Reply:
@jhnvdw could you help me understand how we're avoiding duplicating data bw intraday + daily here?

Reply:
Well, it should not matter if data is coming from the one table or another, as long as collected events are unique for a user and event timestamp we can make sure there won't be double counting events. All events happen in the context of a session, so deduplicating events will also deduplicate sessions.

Please let me know if I'm missing something here.

Reply:
To show what I'm proposing I've merged a new PR in my fork (https://github.com/jhnvdw/ga4-bigquery-starter/pull/4/files). I can't push it here because the PR in this thread is still open.